graphbolt/src/csc_sampling_graph.cc

```diff
@@ -178,15 +306,10 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighborsImpl(
                   continue;
                 }
 
-                if (consider_etype) {
-                  picked_neighbors_cur_thread[i - begin] = PickByEtype(
-                      offset, num_neighbors, fanouts, replace, indptr_options,
-                      type_per_edge_.value(), probs_or_mask, args);
-                } else {
-                  picked_neighbors_cur_thread[i - begin] = Pick(
-                      offset, num_neighbors, fanouts[0], replace,
-                      indptr_options, probs_or_mask, args);
-                }
+                picked_neighbors_cur_thread[i - begin] =
+                    pick_fn(offset, num_neighbors);
```

Comment:
That's ok for this PR, but in successor change, `pick_fn` should receive concert number of picked number as argument, as it is already calculated in the `num_pick_fn`.